api/openapi-spec/swagger.json

```diff
@@ -46070,35 +46070,6 @@
       "required": true
      }
     ]
-   },
-   "/version/": {
```

Comment:
> Can you elaborate on the problem this causes for aggregation?

Subpaths under `/version` are hidden when chaining servers together.  It means that you can't have `/version/apiextensions-apiserver` or `/version/openshift`.